.infra/clickhouse-sync.yml

```diff
@@ -0,0 +1,131 @@
+offset.storage.jdbc.password: "%clickhouse_password%"
+
+# offset.storage.jdbc.offset.table.ddl: The DDL statement used to create the database table where connector offsets are to be stored.
+offset.storage.jdbc.offset.table.ddl: |
+  CREATE TABLE if not exists %s
+  (
+      `id` String,
+      `offset_key` String,
+      `offset_val` String,
+      `record_insert_ts` DateTime,
+      `record_insert_seq` UInt64,
+      `_version` UInt64 MATERIALIZED toUnixTimestamp64Nano(now64(9))
+  )
+  ENGINE = ReplacingMergeTree(_version)
+  ORDER BY id
+  SETTINGS index_granularity = 8198
+offset.storage.jdbc.offset.table.delete: "delete from %s where 1=1"
+schema.history.internal: "io.debezium.storage.jdbc.history.JdbcSchemaHistory"
+schema.history.internal.jdbc.url: "jdbc:clickhouse://%clickhouse_host%:%clickhouse_port%/altinity_sink_connector"
+schema.history.internal.jdbc.user: "%clickhouse_user%"
+schema.history.internal.jdbc.password: "%clickhouse_password%"
+schema.history.internal.jdbc.schema.history.table.ddl: |
+  CREATE TABLE if not exists %s
+  (
+    `id` VARCHAR(36) NOT NULL,
+    `history_data` VARCHAR(65000),
+    `history_data_seq` INTEGER,
+    `record_insert_ts` TIMESTAMP NOT NULL,
+    `record_insert_seq` INTEGER NOT NULL
+  )
+  ENGINE = ReplacingMergeTree(record_insert_seq)
+  order by id
+
+# schema.history.internal.schema.history.table.name: The name of the database table where connector schema history is to be stored.
+schema.history.internal.jdbc.schema.history.table.name: "altinity_sink_connector.replicate_schema_history"
+
+# enable.snapshot.ddl: If set to true, the connector wil parse the DDL statements as part of initial load.
+enable.snapshot.ddl: "true"
+
+# auto.create.tables: If set to true, the connector will create the database tables for the destination tables if they do not already exist.
+auto.create.tables: "true"
+
+# database.dbname: The name of the PostgreSQL database from which events are to be captured when not using snapshot mode.
+database.dbname: "%database_dbname%"
+
+# clickhouse.datetime.timezone: This timezone will override the default timezone of ClickHouse server. Timezone columns will be set to this timezone.
+#clickhouse.datetime.timezone: "UTC"
+
+# skip_replica_start: If set to true, the connector will skip replication on startup. sink-connector-client start_replica will start replication.
+skip_replica_start: "false"
+
+# binary.handling.mode: The mode for handling binary values. Possible values are bytes, base64, and decode. The default is bytes.
+#binary.handling.mode: "base64"
+
+# ignore_delete: If set to true, the connector will ignore delete events. The default is false.
+#ignore_delete: "true"
+
+#disable.ddl: If set to true, the connector will ignore DDL events. The default is false.
+#disable.ddl: "false"
+
+#disable.drop.truncate: If set to true, the connector will ignore drop and truncate events. The default is false.
+#disable.drop.truncate: "false"
```

Comment:
These you still need here?

Reply:
We could come back once deployed and all good and cleanup commented out config lines?

Reply:
can we 🙏 keep config readable? just to avoid pain of finding it again if needed.

Reply:
I don't mind either way